package/src/contexts/messagesContext/MessagesContext.tsx

```diff
@@ -296,6 +296,14 @@ export type MessagesContextValue<
   handleRetry?: (message: MessageType<StreamChatGenerics>) => Promise<void>;
   /** Handler to access when a thread reply action is invoked */
   handleThreadReply?: (message: MessageType<StreamChatGenerics>) => Promise<void>;
+  isAttachmentEqual?: (
+    prevProps: Attachment<StreamChatGenerics>,
```

Comment:
lets rename it to `prevAttachment` and `nextAttachment` to make it explicit. Same for message. - `prevMessage` and `nextMessage`